dialout/dialout_client/dialout_client_test.go

```diff
@@ -503,6 +507,10 @@ func TestGNMIDialOutPublish(t *testing.T) {
 }
 
 func init() {
+	// Enable logs at UT setup
+	flag.Lookup("v").Value.Set("10")
```

Comment:
Is this needed for master?

Reply:
Not necessarily, removing. But I think this could be useful to mention in readme/onenote as a debugging help.